tests/test_projectbuilder.py

```diff
@@ -136,6 +136,18 @@ def test_check_dependency(monkeypatch, requirement_string, expected):
     assert next(build.check_dependency(requirement_string), None) == expected
 
 
+def test_bad_project(test_no_project_path):
+    # Passing a nonexistent project directory
+    with pytest.raises(build.BuildException):
```

Comment:
Can you split each test into its own test please? 👌

Reply:
The code already went in. This is not a big deal though.